src/empty-state-card.tsx

```diff
@@ -18,19 +19,30 @@ const useStyles = createUseStyles((theme) => ({
     },
     image: {
         height: 80,
+        display: 'block', // to avoid letter's descenders bottom space
         [theme.mq.tabletOrSmaller]: {
             height: 64,
         },
     },
+    iconContainer: {
+        width: 80,
+        height: 80,
+        [theme.mq.tabletOrSmaller]: {
+            width: 64,
+            height: 64,
+        },
+    },
     actions: {
         marginLeft: ({needsButtonLinkAlignment}) => (needsButtonLinkAlignment ? -6 : 0),
     },
 }));
 
 interface CommonProps {
     title: string;
```

Comment:
Empty state use different text style than Empty State Card. Please, check specs https://zpl.io/a3E4GMY

Reply:
oops! done! :fearful:

Reply:
> * [ ]  TODO: check emptyState comp inside master-detail layout: @yceballost what do you thing about this? [bit.ly/2WQjVZo](https://bit.ly/2WQjVZo)

We can use EmptyStateCard by the moment, there aren't cases with this problem right now, so.. imo, I dont try to solve it now